Sources/SwiftSyntax/generated/SyntaxBaseNodes.swift

```diff
@@ -90,49 +84,55 @@ extension DeclSyntaxProtocol {
   /// Force-upcast the current syntax node to its base node type (``DeclSyntax``).
   ///
   /// - Returns: The base node created from the current syntax node, as the node can always be upcast to its base type.
-  ///
-  /// - Note: This method overloads the general `as` method and is marked deprecated to produce a warning
-  ///         informing the user the upcast should be performed using the target base node's initializer.
   @available(*, deprecated, message: "Use `DeclSyntax.init` for upcasting")
   public func cast(_ syntaxType: DeclSyntax.Type) -> DeclSyntax {
     return DeclSyntax(self)
   }
   
 
-  /// Checks if the current syntax node can be cast to a given node type from the different base node protocol hierarchy than ``DeclSyntaxProtocol``.
+  /// Checks if the current syntax node can be cast to a given node type from a base node protocol hierarchy other
+  /// than ``DeclSyntaxProtocol``.
   ///
-  /// - Returns: `false` since the node can not be cast to the node type from different base node protocol hierarchy than ``DeclSyntaxProtocol``.
+  /// - Returns: `true` if the node can be cast, `false` otherwise.
   ///
-  /// - Note: This method overloads the general `is` method and is marked as deprecated to produce a warning,
-  ///         informing the user that the cast will always fail.
-  @available(*, deprecated, message: "This cast will always fail")
+  /// - Note: This generally does not make sense to call - if in a generic ``SyntaxProtocol`` context, use
+  /// ``DeclSyntaxProtocol`` instead. Otherwise this must be a node outside of ``DeclSyntaxProtocol``,
+  /// in which case this call will always fail.
+  @available(*, deprecated, message: "Type should be part of the 'DeclSyntaxProtocol' hierarchy")
   public func `is`<S: SyntaxProtocol>(_ syntaxType: S.Type) -> Bool {
-    return false
+    return self.as(syntaxType) != nil
   }
   
 
-  /// Attempts to cast the current syntax node to a given node type from the different base node protocol hierarchy than ``DeclSyntaxProtocol``.
+  /// Attempts to cast the current syntax node to a given node type from the a base node protocol hierarchy other than
+  /// ``DeclSyntaxProtocol``. This is generally does not make sense to call - if in a generic ``SyntaxProtocol``
+  /// context, use ``DeclSyntaxProtocol`` instead. Otherwise this must be a base node other than
+  /// ``DeclSyntaxProtocol``, in which case this cast will always return `nil`.
```

Comment:
I think this is duplicating the `- Note:`

Reply:
I swear I removed this 🤔. Thanks